seekinternational/notification-library
# Feature/unit test email

What does this PR do?
---------------------
add unit test for the happy path of send email
corrected typos

How can I test this?
--------------------
1. Run `npm test`
2. Check for unit test coverage of email. Should be above 90%

Notes
-----
<Enter notes, remove if not applicable>
